beginner_source/maskedtensor_missing_nan_ops.rst

```diff
@@ -0,0 +1,32 @@
+Implementing missing torch.nan* operators
+-----------------------------------------
+
+In the above issue, there is a request to add additional operators to cover the various `torch.nan*` applications,
+such as ``torch.nanmax``, ``torch.nanmin``, etc.
+
+In general, these problems lend themselves more naturally to masked semantics, so instead of introducing additional
+operators, we propose using MaskedTensors instead. Since
+`nanmean has already landed <https://github.com/pytorch/pytorch/issues/21987>`__, we can use it as a comparison point:
+
+    >>> x = torch.arange(16).float()
+    >>> y = x * x.fmod(4)
+    >>> y = y.masked_fill(y ==0, float('nan'))
+    >>> y
+    tensor([nan,  1.,  4.,  9., nan,  5., 12., 21., nan,  9., 20., 33., nan, 13.,
+            28., 45.])
+    >>> y.nanmean()
+    tensor(16.6667)
+    >>> torch.mean(masked_tensor(y, ~torch.isnan(y)))
```

Comment:
have we considered API sugar:

(1) instantiating a MT from a Tensor assuming na is the mask

```python
>>> MaskedTensor(y)
MaskedTensor(
  [      --,   1.0000,   4.0000,   9.0000,       --,   5.0000,  12.0000,  21.0000,       --,   9.0000,  20.0000,  33.0000,       --,  13.0000,  28.0000,  45.0000]
)
```

(2) instantiating a MT where user just states the mask value instead of passing the mask

```python
y = MaskedTensor(y, mask_value=float(1))
```

Reply:
Not yet! I think an unspecified mask could also be an indication that they would like all `True` values for the mask, so that could be a third option as well.

Another one would be to allow for just `MaskedTensor(y)` if `y` is a sparse tensor because then the mask is "implied".

All been discussed and will take note to add in :)

Reply:
where are you tracking feature requests?